reader-osm/src/test/java/com/graphhopper/reader/osm/GraphHopperOSMTest.java

```diff
@@ -1117,4 +1117,56 @@ private GraphHopper createGraphHopper(String vehicles) {
                 setEncodingManager(em).
                 setProfiles(profiles);
     }
+
+    @Test
+    public void testLoadingWrongCHProfiles() {
```

Comment:
This is probably better in GraphHopperTest but mocking DataReader is a bit lengthy/ugly.

Reply:
Ok ;)